sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -57,10 +57,7 @@ class lowergamma(sym.lowergamma):
 Expressions whose pretty-printing is tested here:
 (A '#' to the right of an expression indicates that its various acceptable
 orderings are accounted for by the tests.)
-
```

Comment:
Please restore this file and keep the empty lines. Will make it easier to read.

If you get errors it is probably because there are spaces on otherwise empty lines (caused by automatic indenting of the editor).